backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Core/Models/SingleRight/Details.cs

```diff
@@ -23,6 +23,6 @@ public class Details
         ///     Further details about the reason for the response.
         /// </summary>
         [JsonProperty("parameters")]
-        public List<Parameters>? Parameters { get; set; }
+        public Parameter? Parameters { get; set; }
```

Comment:
Tried that at first but it doesn't work since the json we'll get from backend is an object and not a list

<img width="476" alt="image" src="https://github.com/Altinn/altinn-access-management-frontend/assets/31471142/3562287d-c7cd-4be4-b158-a8ea89a7e9a3">

Reply:
Aha, ok, then I guess naming the type Parameters will have to suffice